backend/server.js

```diff
@@ -42,14 +41,16 @@ const hapi = new Hapi.Server({
 sbp('okTurtles.data/set', SERVER_INSTANCE, hapi)
 
 sbp('sbp/selectors/register', {
+  'backend/server/broadcastEntry': async function (entry: GIMessage) {
+    const pubsub = sbp('okTurtles.data/get', PUBSUB_INSTANCE)
+    const pubsubMessage = createMessage('entry', entry.serialize())
```

Comment:
Should this `'entry'` be a constant that's referenced from somewhere instead of a string literal?

Reply:
The string literal was less code overall, but we can use an imported constant for consistency.